Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj

```diff
@@ -278,20 +330,21 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
 				PROVISIONING_PROFILE_SPECIFIER = "";
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
Should we use Swift 5 as the default?

Reply:
Swift 5 has been out for over a year so I think so, I don't think it has any source breaking changes between version.

Reply:
We can upgrade to Swift 5, but we should compile and test every project again to validate.